app/ui-react/syndesis/src/app/App.tsx

```diff
@@ -199,56 +210,87 @@ export class App extends React.Component<IAppBaseProps, IAppBaseState> {
-                                  pictograph={
-                                    <img
-                                      src={logo}
-                                      alt="Syndesis"
-                                      style={{ minWidth: '164px' }}
-                                    />
-                                  }
-                                  logoHref={'/'}
-                                  showNavigation={this.state.showNavigation}
-                                  onNavigationCollapse={this.hideNavigation}
-                                  onNavigationExpand={this.showNavigation}
-                                >
-                                  <Notifications
-                                    notifications={this.state.notifications}
-                                    notificationTimerDelay={8000}
-                                    removeNotificationAction={
-                                      this.removeNotificationAction
-                                    }
-                                  />
-                                  <WithRouter>
-                                    {({ match }) => (
-                                      <WithErrorBoundary key={match.url}>
-                                        <Switch>
-                                          {this.props.routes.map(
-                                            (
-                                              { to, exact, component },
-                                              index
-                                            ) => (
-                                              <Route
-                                                path={to}
-                                                exact={exact}
-                                                component={component}
-                                                key={index}
-                                              />
-                                            )
-                                          )}
-                                          <Route component={PageNotFound} />
-                                        </Switch>
-                                      </WithErrorBoundary>
+                            {functions => {
+                              const productName = 'Syndesis';
+                              return (
+                                <ServerEventsContext.Provider value={functions}>
+                                  <AboutModal
+                                    trademark={'Red Hat'}
+                                    productName={productName}
+                                    isModalOpen={this.state.showAboutModal}
+                                    handleModalToggle={this.toggleAboutModal}
+                                    brandImg="https://avatars0.githubusercontent.com/u/23079786"
+                                  >
+                                    {this.state.showAboutModal && (
```

Comment:
This line should go up in front of the `<AboutModal>` tag there on line 217 I think.